### DISCUSSION TITLE
How to commit changes to cmake files?

### DISCUSSION THREAD

**Discussion by Zeblote** [NVIDIA-Omniverse/PhysX#48] [Q&A]
I'm trying to modify some of the build files like I did before with PhysX 4, but they are now located in a mysterious path `packages\chk\CMakeModules\1.28.trunk.31965103\GetCompilerAndPlatform.cmake`.

How do I commit a change to this file if it lives outside the main repository? I'm not sure how to bypass this package downloader.

**Comment by ayoub-belarbi** [comment]
Hello @Zeblote and thank you for contributing to PhysX!
We are now using a dependency manager called Packman to serve all externals/packages instead of storing them inside the repo itself.
Some important notes:

- You can of course change the code in packages\chk\CMakeModules\1.28.trunk.31965103\GetCompilerAndPlatform.cmake (or any other dependency) and it will be picked up by PhysX, of course, the change will be local to your system. However, if the package in question was updated, Packman will download the new package and overwrite any local changes that you made.
- To request some changes to a specific package, please open an issue/discussion thread describing the issue/improvement and we will gladly update the package in the next release.

Can you share here what specific PRs in PhysX 4 for CMakeModules you would like to merge?

Regards,
Ayoub

**Reply by moraaar** [reply]
Thanks @ayoub-belarbi I've created 2 new issues requesting changes to internal packman packages:
- https://github.com/NVIDIA-Omniverse/PhysX/issues/59
- https://github.com/NVIDIA-Omniverse/PhysX/issues/60
